services/sensorCachingService.js

```diff
@@ -0,0 +1,84 @@
+var sensors = {}
+
+/**
+ *Retrieve the last known temprature/humidity form a specific sensor
+ * @author Lars
+ * @param {entry} entry The sensor entry that you want to add to the cache
+ */
+function retrieveLastValue(entry) {
+    const key = generateCacheKey(entry);
+    if(sensors[key]){
+        return sensors[key].value
+    }
+    return null
+}
+
+/**
+ * Store the current value in the sensors obj
+ * @author Lars
+ * @param {entry} entry The sensor entry that you want to add to the cache
+ */
+function storeCurrentValue(entry){
+    sensors[generateCacheKey(entry)] = entry
+}
+
+/**
+ * Retrieve the maximum temprature/humidity for a specific floor.
+ * @author Lars
+ * @param {floor} floor The floor you want to recieve the max temp of.
```

Comment:
here in the curly braces {} should be the data type of the parameter. Now I don't know what conventions are there but it seems reasonable to sue typescript's so that would be {number}
```suggestion
 * @param {number} floor The floor you want to recieve the max temp of.
```